pkg/proxy/iptables/proxier.go

```diff
@@ -298,7 +298,7 @@ func NewProxier(ipt utiliptables.Interface,
 	nodePortAddresses = ipFamilyMap[ipFamily]
 	// Log the IPs not matching the ipFamily
 	if ips, ok := ipFamilyMap[utilproxy.OtherIPFamily(ipFamily)]; ok && len(ips) > 0 {
-		klog.Warningf("IP Family: %s, NodePortAddresses of wrong family; %s", ipFamily, strings.Join(ips, ","))
+		klog.InfoS("IPs not matching the ipFamily", "IP Family", ipFamily, "NodePortAddresses", strings.Join(ips, ","))
```

Comment:
"found node IPs of the wrong family", with keys "ipFamily" or "family" and "ips"